projects/authentication/docs/user-access-token.md

```diff
@@ -271,6 +279,39 @@ To learn more about the Authorization Code Flow with PKCE, see the [the Auth0 do
 >
 > If you want, you can use the [Single-Page Application (SPA) SDK Libraries](https://auth0.com/docs/libraries#spa) provided by Auth0 to implement this flow in frontend Javascript applications. Native applications can use the [Native and Mobile Application SDK Libraries](https://auth0.com/docs/libraries#native).
 
+## Login parameters
+
+When you redirect your user to the `/authorize` endpoint on the authorization server to login, you must provide some required URL parameters and may also use some optional ones for customization. 
+
+The following table gives an overview of all the required/optional parameters:
+
+
+| Parameter name | Possible values | Required? | Description
+|---------|----------|---------|---------
+| `audience` | `https://api.publiq.be` | **Required** | Determines which APIs your token will be usable on. Because Auth0 only supports one audience per token, we use a generic URL single for all our different APIs. Note that in reality your token will only be usable on the APIs that your client has access to.
+| `response_type` | `code` | **Required** | Indicates which OAuth2 grant type will be used. **Must** be set to `code` for the "Authorization Code" flows documented on this page.
+| `client_id` | Your application's client id | **Required** | Used to validate the authorization code when exchanging it for a user access token.
+| `scope` | A space-delimited string with one or more of the following values: `openid`, `profile`, `email`, `offline_access` | **Required** | Determines which user info will be shared with your application, and if you want a refresh token or not. Suggested to always be set to `openid profile email offline_access` for best results.
+| `redirect_uri` | For example `https://your-application.com/authorize` | **Required** | A URL of your application starting with `https://` to redirect the user back to after logging in. A `code` query parameter with the authorization code that can be exchanged for a user access token by your application will be appended at the end.
+| `code_challenge` | String generated from a cryptographically-random code verifier. | Required if using **PKCE** | See the [PKCE example](#example-1) for more info.
+| `code_challenge_method` | `S256` | Required if using **PKCE** | **Must** be set to `S256`. See the [PKCE example](#example-1) for more info.
+| `prompt` | `none` (default) or `login` | Optional | If set to `login`, the user will always be asked to login even if they still have an active session on the authorization server. Useful to force the user to login again, which also makes it possible to switch to another account as before.
+| `referrer` | `museumpas`, `uitdatabank` or `uit` | Optional | A publiq brand, which is used to set the background image of the login page accordingly.
```

Comment:
`uitpas` is probably also a possible value?

Reply:
Ah yes, and `uitdatabank` should actually be `udb` now that I look at it again in the Confluence page where this was documented before. And there is also `cultuurkuur`. I'll fix it!

Reply:
I see why I made these mistakes, I accidentally documented the values for the same `referrer` parameter on the old custom `/login-from-app` that is documented on the same Confluence page

Reply:
Fixed now!